app/models/miq_expression.rb

```diff
@@ -1026,22 +1026,22 @@ def self.evaluate(expression, obj, _inputs = {}, tz = nil)
     result
   end
 
-  def evaluate(obj, inputs = {}, tz = nil)
-    self.class.evaluate(self, obj, inputs, tz)
+  def evaluate(obj, tz = nil)
+    self.class.evaluate(self, obj, tz)
```

Comment:
If the build didn't fail owing to the above do we need to increase our test coverage here?